db/migrate/20200707152500_rename_verification_status_to_processing_status.rb

```diff
@@ -0,0 +1,5 @@
+class RenameVerificationStatusToProcessingStatus < ActiveRecord::Migration[6.0]
+  def change
+    rename_column :wage_verifications, :verification_status, :processing_status
```

Comment:
We want a separate column for this - we still want to store the truework status, the processing status is for our internal use.

Reply:
Code should be:

`add_column :wage_verifications, :processing_status, :integer, limit: 2, default: 0, null: false`

Reply:
also, rename file.